src/EditorFeatures/Core/Diagnostics/AbstractDiagnosticsTaggerProvider.SingleDiagnosticKindPullTaggerProvider.cs

```diff
@@ -132,6 +132,13 @@ protected sealed override ITaggerEventSource CreateEventSource(ITextView? textVi
                     includeSuppressedDiagnostics: true,
                     cancellationToken: cancellationToken).ConfigureAwait(false);
 
+                // Include Copilot diagnostics when computing analyzer semantic diagnostics.
```

Comment:
this comment doesn't tell me anything.  it just says what the code is doing.  but it doesn't tell me *why*.  for example, why do we not get copilot diags for other diagnostic kinds?